core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -79,18 +79,18 @@ public class TaskManagerConfig
     private Duration interruptStuckSplitTasksDetectionInterval = new Duration(2, TimeUnit.MINUTES);
 
     private boolean scaleWritersEnabled = true;
-    // Set the value of default max writer count to the number of processors and cap it to 32. We can do this
+    // Set the value of default max writer count to the number of processors * 2 and cap it to 64. We can do this
     // because preferred write partitioning is always enabled for local exchange thus partitioned inserts will never
     // use this property. Hence, there is no risk in terms of more numbers of physical writers which can cause high
```

Comment:
`there is no risk in terms of more numbers of physical writers which can cause high resource utilization`
I'm not sure this line is true. Every orc/parquet writer has to buffer 64/128MB in memory before flushing to disk. So sounds like more writers results in more resource usage.